components/family-studies/ui/src/main/resources/PhenoTips/FormMacros.xml

```diff
@@ -76,5 +76,22 @@
 #end
 )))
 #end
+
+#macro (__displayFamilyMembers $family)
```

Comment:
This is not the right place for this macro. `FormMacros` is supposed to be a container of just macros used for displaying a generic form, and not something specific to families or patients. Create a new `FamilyMacros` instead.